### ISSUE TITLE
Capability: Move the map to display a given location

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/HTML-Map-Element-UseCases-Requirements#157]
This issue is for discussion of <a href="https://maps4html.github.io/HTML-Map-Element-UseCases-Requirements/#capability-move-the-map-to-display-a-given-location">the required API capability “Move the map to display a given location”</a>.

**Comment by NickFitz** [comment]
I'm wondering whether this capability should be subsumed into the more general [Pan, zoom and re-centre](https://maps4html.org/HTML-Map-Element-UseCases-Requirements/#capability-pan-zoom-or-re-centre-map) capability?

**Comment by zcorpan** [comment]
Yeah this seems to be a duplicate of 6.2.2 Pan, zoom, or re-centre a map unless I'm missing something.
